lib/pull.ts

```diff
@@ -358,6 +461,8 @@ async function downloadAndSave(
       });
     }
 
+    // TODO: update this so that all of the separate component library files get spread under one
+    // key, maintaining backwards compatibility for downstream SDKs
```

Comment:
Is this TODO meant to be here

Reply:
Yes, will be implemented in follow-up PR in the integration branch